packages/marshal/src/marshal.js

```diff
@@ -81,7 +81,7 @@ export const makeMarshal = (
   /**
    * @type {Serialize<Slot>}
    */
-  const serialize = root => {
+  const serialize = (root, inputValidator) => {
```

Comment:
```suggestion
  const serialize = (root, inputValidator = undefined) => {
```

Reply:
I generally like to see optionality explicitly in the parameter list declaration. When the types are not local, this is even more important.